README.md

```diff
@@ -1,13 +1,21 @@
-# Daml Platform - Observability Example
+# [Daml Enterprise](https://www.digitalasset.com/products/daml-enterprise) - Observability Example
 
-This repo demonstrates how you can set up your Daml deployment to leverage the observability
-features available, along with a few example Grafana dashboards for monitoring Daml services.
+This project demonstrates a [Daml
+Enterprise](https://www.digitalasset.com/products/daml-enterprise)
+deployment's observability features, along with example Grafana dashboards.  
+
+The project is self contained:  providing load generations scripts to generate
```

Comment:
I would not call the scripts we have here `load generation scripts`, as it could imply that the scripts can be used to "load test" canton, while the scripts we have just generate demo data.

Reply:
Changed the phrase "load generation scripts."